### DISCUSSION TITLE
Question on "relax" mod related scores bug.

### DISCUSSION THREAD

**Discussion by dolkilu** [ppy/osu#19176] [Q&A] [answered]
Since lazer leaderboard is now available, I have discovered that during a period starting from March 15th, relax scores were submitted as no mods, I wanted ask if this bug is already fixed.
It makes many 6 digits players who are most likely legit look like relax cheaters. This issue maybe related to [#17718](https://github.com/ppy/osu/issues/17718) and [#17319](https://github.com/ppy/osu/discussions/17319).

Examples
![osu_2022-07-17_11-12-28](https://user-images.githubusercontent.com/16611027/179384494-c05d1d66-bc20-4915-969f-db8a706c537a.png)
![osu_2022-07-17_11-21-12](https://user-images.githubusercontent.com/16611027/179384498-3cc27534-e9ed-4558-b053-424384997a3f.png)
![osu_2022-07-17_12-54-07](https://user-images.githubusercontent.com/16611027/179384499-d64131e7-b540-4610-990c-968c48984444.png)
![osu_2022-07-17_12-54-25](https://user-images.githubusercontent.com/16611027/179384501-023dff70-5860-4584-b3e1-c8259fd3b889.png)

**Comment by frenzibyte** [comment, marked as answer]
I'm not sure how you can tell those scores are made with "Relax" mod set on them? Also the issue threads you linked have no connection with this issue, I'm seeing scores with no mods here.

**Reply by dolkilu** [reply]
These maps are very difficult (800pp range), and these scores are done by 6 digits players with 200pp as top plays. However, replays are unavailable so I can't proof that they are just submission errors.
I made this hypothesis because I found that, player "gwav" from 4th screenshot, has a recent relax score on other maps, which made me believe relax scores were incorrectly submitted.

**Reply by dolkilu** [reply]
Another example would be player "NotSavox" from 2nd and 3rd screenshot, all his recent scores are relax mod. Which makes me believe he didn't cheat those 8* scores.
![2022-07-17 01-34-27](https://user-images.githubusercontent.com/16611027/179385370-2df1bc02-4f7a-4441-86ad-94476d60d773.png)

**Reply by jamesbt365** [reply]
As far as I know this was when the aim assist mod came out EVERY score was submitted as "no mod" meaning aim assist and relax combined did broken stuff (now I got the deceit and preon ss) it should be fixed now

**Reply by frenzibyte** [reply]
> As far as I know this was when the aim assist mod came out EVERY score was submitted as "no mod" meaning aim assist and relax combined did broken stuff (now I got the deceit and preon ss) it should be fixed now

Replying to this, that's news to me. I've never seen or heard about scores with two mods being submitted as NM, it was only about those scores accounting to user statistics and nothing else.
